### PR TITLE
Add conversion of PCC and Ability files

### PR DESCRIPTION
**Pull request by alexperovich**

### CODE REVIEW

File: Data/core_rulebook/cr_abilities.lua [hunk 1]
```diff
@@ -0,0 +1,10580 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr_abilities.lst
+-- From repository https://github.com/pcgen/pcgen at commit 11ceb52482855f2e5f0f6c108c3dc665b12af237
+SetSource({
+  SourceLong="Core Rulebook",
+  SourceShort="CR",
+  SourceWeb="http://paizo.com/store/downloads/pathfinder/pathfinderRPG/v5748btpy88yj",
+  SourceDate="2009-08",
+})
+ModifyAbility({
+  Name="Default",
```
Comment (ChadNedzlek) [inline Data/core_rulebook/cr_abilities.lua:10]: I assume this is a magic ability that just gets applied to everyone?

File: Data/core_rulebook/cr_abilities.lua [hunk 2]
```diff
@@ -0,0 +1,10580 @@
...
+    {
+      Category="VAR",
+      Variables={
+        "DwarvenWaraxeExoticUse",
+      },
+      Formula="1",
+      Conditions={
+        function (character)
+          return 1 <= #filter(character.Abilities, function (ability)
+            return ability.Category == "FEAT" and (ability.Name == "Exotic Weapon Proficiency (Waraxe (Dwarven))")
```
Comment (ChadNedzlek) [inline Data/core_rulebook/cr_abilities.lua:93]: This feels _really_ weird and backward... why doesn't the feat just set this variable?  Everything is Yoda defined?  This ability is zero, unless you have this feat which... just exists and has no other mechanical impact?

File: Data/core_rulebook/cr_abilities.lua (hunk 1 above)
Comment (alexperovich) [reply Data/core_rulebook/cr_abilities.lua:10]: I kinda assume so, but I'm not sure. Its just whats in the lst files.

File: Data/core_rulebook/cr_abilities.lua (hunk 2 above)
Comment (alexperovich) [reply Data/core_rulebook/cr_abilities.lua:93]: Yea, I have no idea, there are some really odd things in the lst files.

File: Data/core_rulebook/cr_abilities.lua (hunk 1 above)
Comment (alexperovich) [reply Data/core_rulebook/cr_abilities.lua:10]: Aha, I found it. The original definition of the "Default" ability is in `ce_abilities_race.lst` I have no idea why its there, but there is a comment about it too.

File: Data/core_rulebook/cr_abilities.lua [hunk 3]
```diff
@@ -0,0 +1,10427 @@
...
+  Stackable=false,
+  Bonuses={
+    {
+      Category="VAR",
+      Variables={
+        "ShieldBashingDieSizeStep",
+      },
+      Formula="3",
+      Conditions={
+        function (character)
```
Comment (ChadNedzlek) [inline Data/core_rulebook/cr_abilities.lua:4490]: This is _really_ verbose way of saying "the character is small, and they have a light shield", yes?  Is there any way to condense it at all?

Comment (ChadNedzlek) [reply Data/core_rulebook/cr_abilities.lua:4490]: Honestly... this seems like this should just be a weapon, that would solve it right quick. :-) But I guess that's not how they implemented it *sigh*

Comment (alexperovich) [reply Data/core_rulebook/cr_abilities.lua:4490]: Yea, this is very weird. this function is a `PREMULT:2,[PRESIZE:M],[PREEQUIP:Type=Shield.Light]`, which is basically a very verbose "and".
This is something we can translate after getting everything setup.

Comment (alexperovich) [reply Data/core_rulebook/cr_abilities.lua:4490]: And... FFS, I think thats another bug, I think that looks for something equipped that is both "Type=Shield" and "Type=Light", so I need to parse out the period separated types.....

**Pull request merged**